TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -527,7 +527,16 @@ private void RefreshCustomSymbols()
             try
             {
                 _customSymbols = Tools.APIHelpers.Helpers.LoadIntrinsic(TypeCobolConfiguration.Copies, TypeCobolConfiguration.Format, DiagnosticsErrorEvent); //Refresh Intrinsics
-                _customSymbols = Tools.APIHelpers.Helpers.LoadDependencies(TypeCobolConfiguration.Dependencies, TypeCobolConfiguration.Format, _customSymbols, TypeCobolConfiguration.InputFiles, TypeCobolConfiguration.CopyFolders, DiagnosticsErrorEvent); //Refresh Dependencies
+                _customSymbols = Tools.APIHelpers.Helpers.LoadDependencies(TypeCobolConfiguration.Dependencies, TypeCobolConfiguration.Format, _customSymbols, TypeCobolConfiguration.InputFiles, 
+                    TypeCobolConfiguration.CopyFolders, DiagnosticsErrorEvent, out List<RemarksDirective.TextNameVariation> usedCopies, out List<CopyDirective> missingCopies); //Refresh Dependencies
+
+                if(missingCopies.Count > 0)
+                {
+                    MissingCopiesEvent(TypeCobolConfiguration.Dependencies?.FirstOrDefault(), new MissingCopiesEvent() { Copies = missingCopies.Select(c => c.TextName).Distinct().ToList() });
+                    var message = "COPY need to be downloaded. When it's done restart the server";
```

Comment:
Isn't it possible to avoid a server restart.
Missing copies is very common case. It will be very boring to restart the server each time it happens.

Reply:
I forgot to remove the notification. All create/delete/update actions on a copy lead to an automatic symbol reloading